doc/source/ray-overview/installation.rst

```diff
@@ -495,3 +495,27 @@ that you've cloned the git repository.
 .. code-block:: bash
 
   python -m pytest -v python/ray/tests/test_mini.py
+
+
+Installed Python dependencies
```

Comment:
How frequently should this be updated? Should the release manager be updating this?

Reply:
Yes, I think the release manager should update it while preparing a release (for the `latest` documentation). I can add this to the release process doc.